docs/references/javascript/user/user.mdx

```diff
@@ -24,6 +24,7 @@ The ClerkJS SDK provides some helper [methods](#methods) on the `User` object to
 | `username` | `string \| null` | The user's username. |
 | `hasImage` | `boolean` | A getter boolean to check if the user has uploaded an image or one was copied from OAuth. Returns `false` if Clerk is displaying an avatar for the user. |
 | `imageUrl` | `string` | Holds the default avatar or user's uploaded profile image. Compatible with Clerk's [Image Optimization](/docs/guides/image-optimization/imageurl-image-optimization). |
+| `passkeys` | <code>[PasskeyResource](/docs/references/javascript/types/passkey-resource)[]</code> \| `null` | An array of passkeys associated with the user's account. |
```

Comment:
I think this should work, and also doesn't break up the code snippet - lmk if it renders correctly !

```suggestion
| `passkeys` | <code>[PasskeyResource](/docs/references/javascript/types/passkey-resource)[] \| null</code> | An array of passkeys associated with the user's account. |
```